docker-compose.yml

```diff
@@ -2,13 +2,13 @@ version: '3'
 services:
   db:
     image: postgres:10.3
-    ports:
-      - 5432:5432
+    expose:
+      - 5432
```

Comment:
[IMO] I think you can remove `expose` options because of Dockerfile has `EXPOSE` option.
https://github.com/docker-library/postgres/blob/master/10/alpine/Dockerfile#L189
https://hub.docker.com/layers/redis/library/redis/4.0.14-alpine/images/sha256-b73b70e82b0b742ec1a4681b65f2f451128f4bc9c66316e15922392660322d94?context=explore

```suggestion
```

Reply:
This is a good simplification. I've force-pushed a new commit.